Tests/Functional/ContinueAfterFailureTestCase/main.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %{swiftc} %s -o %T/ContinueAfterFailureTestCase
+// RUN: %T/ContinueAfterFailureTestCase > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(macOS)
+    import SwiftXCTest
+#else
+    import XCTest
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'ContinueAfterFailureTestCase' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+class ContinueAfterFailureTestCase: XCTestCase {
+    static var allTests = {
+        return [
+            ("testDoesNotContinueAfterFailure", testDoesNotContinueAfterFailure),
+            ("testContinueAfterFailure", testContinueAfterFailure)
+        ]
+    }()
+    
+    // CHECK: Test Case 'ContinueAfterFailureTestCase.testDoesNotContinueAfterFailure' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+    // CHECK: First Log in DoesNotContinueAfterFailure test
+    // CHECK: .*/ContinueAfterFailureTestCase/main.swift:[[@LINE+4]]: error: ContinueAfterFailureTestCase.testDoesNotContinueAfterFailure : XCTAssertTrue failed -
+    func testDoesNotContinueAfterFailure() {
+        continueAfterFailure = false
+        print("First Log in DoesNotContinueAfterFailure test")
+        XCTAssert(false)
+        print("Second Log in DoesNotContinueAfterFailure test")
+    }
+    
+    func testContinueAfterFailure() {
```

Comment:
There don't seem to be any assertions for this function, did you forget to add them? What's being tested here?

Reply:
I will add also the CHECK-NOT to this part (thanks for the tip), because with the current behavior all the test process crashes and it doesn't print anything for the second test (testContinueAfterFailure). What's being tested right now is that the remaining of the test suite doesn't execute.